src/src/main/java/io/codelex/classesandobjects/practice/Product.java

```diff
@@ -0,0 +1,41 @@
+package io.codelex.classesandobjects.practice;
+
+public class Product {
+    private final String name;
+    private double priceAtStart;
+    private int amountAtStart;
+
+
+    public Product(String name, double priceAtStart, int amountAtStart) {
+        this.name = name;
+        this.priceAtStart = priceAtStart;
+        this.amountAtStart = amountAtStart;
+    }
+
+    public void printProduct() {
+        System.out.println(name + ", price " + String.format("%.2f", priceAtStart) + " EUR, amount " + amountAtStart);
+    }
+
+    public void setPriceAtStart(double priceAtStart) {
+        this.priceAtStart = priceAtStart;
+    }
+
+    public void setAmountAtStart(int amountAtStart) {
+        this.amountAtStart = amountAtStart;
+    }
+
+    public static void main(String[] args) {
```

Comment:
Nu jā, šis pa lielam arī ir diezgan greizi - jo faktiski Tu no klases, kura pati par sevi ir ar main palaižamu metodi taisi vēl objektus. Šai main metodei vajadzētu būt citā failā.